test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -249,7 +249,7 @@ private string GetBuildPrefix()
 #endif
         }
 
-        [Fact]
+        [Fact(Skip = "Execute this test if you want to regenerate the core_manifest and core_ep_list files")]
```

Comment:
>Skip = "Execute this test if you want to regenerate the core_manifest and core_ep_list files") [](start = 14, length = 94)

[https://media.tenor.com/images/f6f91f0f5bd4d733e1fb9dfee1c37580/tenor.gif](https://media.tenor.com/images/f6f91f0f5bd4d733e1fb9dfee1c37580/tenor.gif)

Reply:
I totally AM!

---
In reply to: [232741497](https://github.com/dotnet/machinelearning/pull/1590#discussion_r232741497) [](ancestors = 232741497)